torch/_functorch/partitioners.py

```diff
@@ -1228,6 +1228,7 @@ def get_default_op_list() -> OpTypes:
         aten.expand,
         aten.as_strided,
         aten.permute,
+        aten.select,
```

Comment:
this fixes the `[select_1, select_3, select_5, select_7]` bwd graph input duplication issue.

Reply:
cc @Chillee is there any reason we hardcode a list of view ops here, instead of just checking the current OpOverload's `is_view` attribute? There are probably other view ops here we are missing too.

(or maybe slightly different question: are there any "non-recomputable view ops"?

Reply:
@bdhirsh I've been meaning to change all of these to use tags, just haven't gotten around to it.

There are some view ops I've been a bit worried about, like `aten.split`. But mostly I think they should be fine.

Reply:
yeah sounds great - maybe I can do "checking the tag instead of a hardcoded list" in a separate PR? in case there are other things that we need to fix to make that change happen